doc/api/fs.md

```diff
@@ -3598,6 +3598,51 @@ that represent files will be deleted. The permissive behavior of the
 `recursive` option is deprecated, `ENOTDIR` and `ENOENT` will be thrown in
 the future.
 
+## `fs.rm(path[, options], callback)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `options` {Object}
+  * `force` Ignore errors
+  * `maxRetries` {integer} If an `EBUSY`, `EMFILE`, `ENFILE`, `ENOTEMPTY`, or
+    `EPERM` error is encountered, Node.js will retry the operation with a linear
+    backoff wait of `retryDelay` ms longer on each try. This option represents
+    the number of retries. This option is ignored if the `recursive` option is
+    not `true`. **Default:** `0`.
+  * `recursive` {boolean} If `true`, perform a recursive removal. In
+    recursive mode operations are retried on failure. **Default:** `false`.
+  * `retryDelay` {integer} The amount of time in milliseconds to wait between
+    retries. This option is ignored if the `recursive` option is not `true`.
+    **Default:** `100`.
+* `callback` {Function}
+  * `err` {Error}
+
+Asynchronous rm(2). No arguments other than a possible exception are given
```

Comment:
I _think_ this should be `rm(1)`.

It seems that most docs referencing man page definitions refer to system calls. As this one isn't, maybe it ought to be more formally/fully described?

Reply:
> As this one isn't, maybe it ought to be more formally/fully described?

@bengl @iansu I agree, since the goal hear is to provide an API surface analogous to `rm`, but we're not actually using the underlying `rm` system call, perhaps we could provide a description, perhaps motivated by `rm`, but not referencing the system call directly?

Reply:
The `fs` docs start with this:

> The `fs` module enables interacting with the file system in a way modeled on standard POSIX functions.

Perhaps we can...uh...model the text here on the "modeled on" phrasing above? We can say this method's behavior is "modeled on the POSIX `rm` command" or something like that?